root/work/WorkMerge.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import sortByEntityName
+  from '../static/scripts/common/utility/sortByEntityName';
+import EnterEdit from '../components/EnterEdit';
+import EnterEditNote from '../components/EnterEditNote';
+import FieldErrors from '../components/FieldErrors';
+import WorkList from '../components/list/WorkList';
+import {withCatalystContext} from '../context';
+import Layout from '../layout';
+
+type Props = {
+  +$c: CatalystContextT,
+  +form: MergeFormT,
+  +toMerge: $ReadOnlyArray<WorkT>,
+};
+
+const WorkMerge = ({$c, form, toMerge}: Props) => (
+  <Layout fullWidth title={l('Merge works')}>
+    <div id="content">
+      <h1>{l('Merge works')}</h1>
+      <p>
+        {l(`You are about to merge the following works into a single
+            work. Please select the work which you would
+            like other works to be merged into:`)}
```

Comment:
This changes the string FWIW. It makes it consistent with the other entities, though the old wording actually sounds better to me (this sounds like an article is missing, and the "to be" is awkward). But it's not wrong either, so it might be better to make them consistent, since both versions sound like bad tongue twisters anyway.

Reply:
Oh, I didn't even notice... I actually think the wording is crap too, but I didn't have any good ideas. "You're about to merge all these works into a single one. Please select the work all others should be merged into:" is probably the least bad I can think of.

Reply:
Yeah, I think your suggestion sounds 100x better than what we have, though you don't have to change anything in this PR if you don't want to. :)

Reply:
Undid the string change here, then added a commit standardising all of these to the same more normal-sounding string.